code/modules/projectiles/guns/energy/laser_vr.dm

```diff
@@ -251,15 +254,15 @@
 	icon = 'icons/obj/gun_vr.dmi'
 	icon_state = "holdoutkill"
 	item_state = null
-	phase_power = 100
+	phase_power = 15
 
 	w_class = ITEMSIZE_SMALL
-	charge_cost = 600
+	charge_cost = 200 //Chompedit Reduced cost
 	modifystate = "holdoutkill"
 	//CHOMP Edit: Changed beam type to new phaser beam type.
 	firemodes = list(
-		list(mode_name="lethal", fire_delay=12, projectile_type=/obj/item/projectile/beam/phaser, modifystate="holdoutkill", charge_cost = 600),
-		list(mode_name="low-power", fire_delay=8, projectile_type=/obj/item/projectile/beam/phaser/light, modifystate="holdoutstun", charge_cost = 120),
+		list(mode_name="lethal", fire_delay=20, projectile_type=/obj/item/projectile/beam/phaser, modifystate="holdoutkill", charge_cost = 200), //Chompedit Reduced cost
```

Comment:
What's the reasoning behind the fire delay being increased so much on the pistol? This makes it a poor close-quarters weapon - you usually only have a limited time to use this if you're pulling it out in close ranges, iirc.

Reply:
That was intended to be an overcharge mode, but I forgot to name them, hence longer fire delay, though it might be too long.